StacklebergCompiler/ReservedNames.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StacklebergCompiler
+{
+    public static class ReservedNames
+    {
+        // Prefixes
+        public static readonly string LeaderActionPrefix = "fix_";
+        public static readonly string FollowerActionPrefix = "attack_";
```

Comment:
Is there a reason to fix and attack? Wasn't legacy from before they figured out better terminology

Reply:
These names are the names the Stackelberg Planner uses
We would need to modify the Planner to be able to use other names, and i would rather not do that